test/distributed/org/apache/cassandra/distributed/impl/InstanceConfig.java

```diff
@@ -197,11 +206,14 @@ public void validate()
             throw new IllegalArgumentException("In-JVM dtests do not support vnodes as of now.");
     }
 
-    private void propagate(Object writeToConfig, String fieldName, Object value, boolean ignoreMissing)
```

Comment:
why drop `ignoreMissing`?

Reply:
implemented the same behaviour via mapping..